src/instructlab/clickext.py

```diff
@@ -197,6 +198,10 @@ def consume_value(
         value, source = super().consume_value(ctx, opts)
         # fix parameter source for config section that are mis-reported
         # as DEFAULT source instead of DEFAULT_MAP source.
+
+        if isinstance(value, enum.Enum):
```

Comment:
can we maybe comment why we are doing this? I know you know and I know, but just for others!

Reply:
added